Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/Examples/BypDam.mo

```diff
@@ -0,0 +1,155 @@
+  package Medium2 = Buildings.Media.Air
+    "Medium model for exhaust air";
+  Buildings.Fluid.Sources.Boundary_pT sin_2(
+    redeclare package Medium = Medium2,
+    T=273.15 + 10,
+    use_p_in=true,
+    nPorts=1)
+    "Sink of exhaust air"
+    annotation (Placement(transformation(extent={{-58,-10},
+            {-38,10}})));
+    Modelica.Blocks.Sources.Ramp PIn(
+    height=200,
+    duration=60,
+    offset=101330)
+    "Pressure of exhaust air"
+    annotation (Placement(transformation(extent={{-20,-50},{0,-30}})));
+  Buildings.Fluid.Sources.Boundary_pT sou_2(
+    redeclare package Medium = Medium2,
+    T=273.15 + 5,
+    use_p_in=true,
+    use_T_in=true,
+    nPorts=1)
+    "Source of exhaust air"
+    annotation (Placement(transformation(extent={{40,-70},
+            {60,-50}})));
+    Modelica.Blocks.Sources.Ramp TSup(
+    height=10,
+    duration=60,
+    offset=273.15 + 30,
+    startTime=60)
+    "Supply air temperature"
+    annotation (Placement(transformation(extent={{-100,44},{-80,64}})));
+  Modelica.Blocks.Sources.Constant TExh(k=293.15)
+    "Temperature of exhaust air"
+    annotation (Placement(transformation(extent={{-20,-90},{0,-70}})));
+  Modelica.Blocks.Sources.Constant POut(k=101325) "Ambient pressure"
+    annotation (Placement(transformation(extent={{-100,-2},{-80,18}})));
+  Buildings.Fluid.Sources.Boundary_pT sin_1(
+    redeclare package Medium = Medium1,
+    T=273.15 + 30,
+    X={0.012,1 - 0.012},
+    use_p_in=true,
+    p=300000,
+    nPorts=1)
+    "Sink of supply air"
+    annotation (Placement(transformation(extent={{84,2},{
+            64,22}})));
+  Buildings.Fluid.Sources.Boundary_pT sou_1(
+    redeclare package Medium = Medium1,
+    T=273.15 + 50,
+    X={0.012,1 - 0.012},
+    use_T_in=true,
+    p=100000,
+    nPorts=1)
+    "Source of supply air"
+    annotation (Placement(transformation(extent={{-60,40},
+            {-40,60}})));
+    Modelica.Blocks.Sources.Ramp PSin_1(
+    duration=60,
+    startTime=240,
+    height=100,
+    offset=1E5 - 110)
+    "Pressure of the supply air"
+    annotation (Placement(transformation(extent={{40,60},{60,80}})));
+  Buildings.Fluid.HeatExchangers.AirToAirHeatRecovery.Wheel whe(
+    redeclare package Medium1 = Medium1,
+    redeclare package Medium2 = Medium2,
+    m1_flow(start=5),
+    m2_flow(start=5),
+    m1_flow_nominal=5,
+    m2_flow_nominal=5,
+    dp1_nominal=100,
+    dp2_nominal=100,
+    show_T=true,
+    controlType=Buildings.Fluid.HeatExchangers.AirToAirHeatRecovery.Types.RecoveryControlType.Bypass,
+    epsL_cool_nominal=0.7,
+    epsL_cool_partload=0.6,
+    epsL_heat_nominal=0.7,
+    epsL_heat_partload=0.6)
+    "Wheel"
+    annotation (Placement(transformation(extent={{6,-4},{26,16}})));
+    Modelica.Blocks.Sources.Ramp DamPos(
+    height=1.0,
+    duration=60,
+    offset=0,
+    startTime=60) "Damper position"
+    annotation (Placement(transformation(extent={{-80,-40},{-60,-20}})));
+  inner Modelica.Fluid.System system
```

Comment:
is this instance needed? There is also one in `VarSpe.mo`.

Reply:
@JayHuLBL just saw that you merged this PR, Thanks a lot. 
I was trying to get rid of this instance but got the following message when translating the model:

```
Note: 
Your model is using an outer "system" component but
an inner "system" component is not defined.
For simulation drag Modelica.Fluid.System into your model
to specify system properties.
```

Can you please let me know if there are any fixes to it?